src/bindist/config.sys

```diff
@@ -9,6 +7,6 @@ buffers=10
 devicehigh=e:\dosemu\ems.sys
 devicehigh=e:\dosemu\cdrom.sys
 rem uncomment to load another bitmap font (see also autoemu.bat)
-rem devicehigh=c:\dos\display.sys con=(ega,,1)
+rem devicehigh=d:\dos\display.sys con=(ega,,1)

```

Comment:
Try
country=01,850,C:\DOS\COUNTRY.SYS
in config.sys
Yes, proper locale support can be added
later - will require patching config.sys by
installer.